Volunteering Page

vbnlohani opened the pull request:
Creates volunteering page and links it to footer. Also adds section to list volunteers, receiving the list via back end. Also adds link to homepage in the navbar logo.

Fixes #59. Fixes #49. Fixes #73.

SaptakS commented on pyconpune/templates/volunteer.html:
```diff
@@ -0,0 +1,24 @@
+{% extends 'site_base.html' %}
+    {% block body %}
```
Don't indent django templating code because the final generated html will then have wrong indentation.

SaptakS commented on root/urls.py:
```diff
@@ -10,6 +10,10 @@
             name="homepage"),
         url(r'^coc/$', root.views.coc,
             name="coc"),
+
```
please remove the blank lines here

SaptakS commented on pyconpune/staticfiles/css/main.css:
```diff
@@ -7,15 +7,15 @@
     padding-top: 50px; /* margin-top = header's height */
```
The indentation changes still have many indentation errors. In case you are changing the indentation then make it same all through out. Some places there are 2 spaces, some place it is 4 and some place 8. Please re-check.

SaptakS commented on pyconpune/templates/volunteer.html:
```diff
@@ -0,0 +1,24 @@
+        <section class="volunteer_info no-margin section-padding float-left full-width gray-bg">
+            <div class="col-960 col-center full-height">
+                <div class="volunteer_info">
+                    <h2> Be A Volunteer ... </h2>
```
Remove the dots.

SaptakS commented:
Please squash your commits. Though they address different purposes I agree but there are too many commits for a small change. You can amend the commit message to mention all the changes done.

vbnlohani committed:
1. Adds new volunteering static page.
2. Modifies gitignore to include venv.
3. Adds a new type of button.

vbnlohani commented:
Did the changes as suggested.

sayanchowdhury commented:
This page should also list all the volunteers. So, create a placeholder for the same. You can pick the idea from the speakers in the front page.

Also the footer should be sticky, right now it is broken. [Read this](https://css-tricks.com/couple-takes-sticky-footer/)

vbnlohani committed:
1. Adds section for volunteers list, list is received via backend.
2. Changes volunteer button style.
3. Fixes broken footer.
4. Adds link to homepage in the navbar logo.
